### PR TITLE
[BTFS-794] Add generic db init scripts

### PR DESCRIPTION
**Pull request by wxue**

### CODE REVIEW

File: db/postgres/scripts/Readme.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# prostegress database generic scripts
```
Comment (taiyangc) [inline db/postgres/scripts/Readme.md]: typo

File: db/postgres/scripts/create_db [hunk 2]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```
Comment (taiyangc) [inline db/postgres/scripts/create_db]: call it create_db.sh, to be more specific, or does k8s prefer this way?

File: db/postgres/scripts/drop_db [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```
Comment (taiyangc) [inline db/postgres/scripts/drop_db]: same file name question

File: db/postgres/scripts/create_db (hunk 2 above)
Comment (wxue) [reply db/postgres/scripts/create_db]: thanks, updated. 
-- k8s YAML file can detect either way, but yeah, it's more informative for a developer to do so

**Review by taiyangc — APPROVED** [review]
lgtm

**Pull request merged**